pythainlp/tokenize/thaisumcut.py

```diff
@@ -0,0 +1,343 @@
+        text = text.replace("พรรคเพื่อ", "<for_party>")
+        text = text.replace("แต่เนื่อง", "<rth_teaneung>")
+        text = text.replace("เพื่อทำให้", "เพื่อ<rth_tamhai>")
+        text = text.replace("ทำเพื่อ", "ทำ<rth_for>")
+        text = text.replace("จึงทำให้", "จึง<tamhai>")
+        text = text.replace("มาโดยตลอด", "<madoitalod>")
+        text = text.replace("แต่อย่างใด", "<teayangdaikptam>")
+        text = text.replace("แต่หลังจาก", "แต่<langjak>")
+        text = text.replace("คงทำให้", "<rth_kongtamhai>")
+        text = text.replace("แต่ทั้งนี้", "แต่<tangni>")
+        text = text.replace("มีแต่", "มี<tea>")
+        text = text.replace("เหตุที่ทำให้", "<hedteetamhai>")
+        text = text.replace("โดยหลังจาก", "โดย<langjak>")
+        text = text.replace("ซึ่งหลังจาก", "ซึ่ง<langjak>")
+        text = text.replace("ตั้งโดย", "<rth_tangdoi>")
+        text = text.replace("โดยตรง", "<rth_doitong>")
+        text = text.replace("นั้นหรือ", "<rth_nanhlor>")
+        text = text.replace("ซึ่งต้องทำให้", "ซึ่งต้อง<tamhai>")
+        text = text.replace("ชื่อต่อมา", "ชื่อ<tomar>")
+        text = text.replace("โดยเร่งด่วน", "<doi>เร่งด่วน")
+        text = text.replace("ไม่ได้ทำให้", "ไม่ได้<tamhai>")
+        text = text.replace("จะทำให้", "จะ<tamhai>")
+        text = text.replace("จนทำให้", "จน<tamhai>")
+        text = text.replace("เว้นแต่", "เว้น<rth_tea>")
+        text = text.replace("ก็ทำให้", "ก็<tamhai>")
+        text = text.replace(" ณ ตอนนั้น", " ณ <tonnan>")
+        text = text.replace("บางส่วน", "บาง<rth_suan>")
+        text = text.replace("หรือแม้แต่", "หรือ<rth_meatea>")
+        text = text.replace("โดยทำให้", "โดย<tamhai>")
+        text = text.replace("หรือเพราะ", "หรือ<rth_orbecause>")
+        text = text.replace("มาแต่", "มา<rth_tea>")
+        text = text.replace("แต่ไม่ทำให้", "แต่<maitamhai>")
+        text = text.replace("ฉะนั้นเมื่อ", "ฉะนั้น<rth_moe>")
+        text = text.replace("เพราะฉะนั้น", "เพราะ<rth_chanan>")
+        text = text.replace("เพราะหลังจาก", "เพราะ<rth_langjak>")
+        text = text.replace("สามารถทำให้", "สามารถ<rth_tamhai>")
+        text = text.replace("อาจทำ", "อาจ<rth_tam>")
+        text = text.replace("จะทำ", "จะ<rth_tam>")
+        text = text.replace("และนอกจากนี้", "นอกจากนี้")
+        text = text.replace("อีกทั้งเพื่อ", "อีกทั้ง<rth_for>")
+        text = text.replace("ทั้งนี้เพื่อ", "ทั้งนี้<rth_for>")
+        text = text.replace("เวลาต่อมา", "เวลา<rth_toma>")
+        text = text.replace("อย่างไรก็ตาม", "อย่างไรก็ตาม")
+        text = text.replace("อย่างไรก็ตามหลังจาก", "<stop>อย่างไรก็ตาม<rth_langjak>")
+        text = text.replace("ซึ่งทำให้", "ซึ่ง<rth_tamhai>")
+        text = text.replace("โดยประมาท", "<doi>ประมาท")
+        text = text.replace("โดยธรรม", "<doi>ธรรม")
+        text = text.replace("โดยสัจจริง", "<doi>สัจจริง")
+
+        if "และ" in text:
+            tokens = pythainlp.word_tokenize(text.strip(), keep_whitespace=True)
```

Comment:
Can you use replace `pythainlp.word_tokenize(text.strip(), keep_whitespace=True)` with `word_tokenize(text.strip(), keep_whitespace=True)` by `from pythainlp.tokenize import word_tokenize` at the head file?